program/rust/src/tests/test_upd_price.rs

```diff
@@ -248,7 +248,7 @@ fn test_upd_price() {
         assert_eq!(price_data.valid_slot_, 5);
         assert_eq!(price_data.agg_.pub_slot_, 6);
         assert_eq!(price_data.agg_.price_, 81);
-        assert_eq!(price_data.agg_.status_, PC_STATUS_TRADING);
+        assert_eq!(price_data.agg_.status_, PC_STATUS_UNKNOWN); // in pythnet aggregation is on the same slot so status turns to unknown
```

Comment:
why in pythnet? this whole codebase is now pythnet only right?

Reply:
will remove to prevent confusion